notebook2/notebook/templates/index.html

```diff
@@ -1,28 +1,30 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="{{ 'dark' if session['dark'] }}">
   <head>
     <title>Hail Notebooks</title>
     <link rel="stylesheet" type="text/css" href="/static/style.css">
     <link rel="stylesheet" type="text/css" href="/static/centering.css">
   </head>
-  <body>
-    <div class="center">
-      <h1>Hail Notebooks</h1>
-      <form action="{{ form_action_url }}" method="post">
-        <p>
-          <label for="password">Class Key:</label>
-          <input type="text" name="password" id="password" />
-        </p>
-        <p>
-          <label for="image">Image:</label>
-          <select name="image" id="image">
-            {% for image in images %}
-            <option value="{{ image }}"{{'selected' if image == default else ''}}>{{ image }}</option>
-            {% endfor %}
-          </select>
-        </p>
-        <input type="submit" value="Start a Hail Notebook" />
-      </form>
-    </div>
+  <body id='theme-site'>
+    <div>{% include 'header.html' %}</div>
+    <span id="main">
```

Comment:
It seems weird to use a `span` for something that's not displayed inline, even if the subsequent styling overrides the display to `flex`. Thoughts on using a `div` instead?

Reply:
Yeah, it may be more semantically correct to use div.